alembic/versions/5b57c5409325_pipelines_refactoring_indexes.py

```diff
@@ -0,0 +1,66 @@
+"""
+
+Revision ID: 5b57c5409325
+Revises: 68ce6a306b30
+Create Date: 2023-03-10 12:27:59.043627
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "5b57c5409325"
+down_revision = "68ce6a306b30"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index(
+        op.f("ix_copr_build_targets_copr_build_group_id"),
+        "copr_build_targets",
+        ["copr_build_group_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_pipelines_copr_build_group_id"),
+        "pipelines",
+        ["copr_build_group_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_pipelines_koji_build_group_id"),
+        "pipelines",
+        ["koji_build_group_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_pipelines_test_run_group_id"),
+        "pipelines",
+        ["test_run_group_id"],
+        unique=False,
+    )
```

Comment:
I'm wondering why are these here?
Those columns (`copr_build_group_id`, `koji_build_group_id`, `test_run_group_id`) already have indexes so this should have already been done in some previous migration, right?

Reply:
I was puzzled like you when I was trying to add the index and I found the code already had it but not the db. I thought that probably we had added the index into the code but we had no run the migration script after that.